docs/content/content/archetypes.md

```diff
@@ -11,140 +11,316 @@ weight: 50
-name = ""
-bio = ""
-genre = ""
+manufacturer = ""
+model = ""
+price = ""
+releaseDate = ""
+rating = ""
++++
+```
+
+But reviews tend to follow strict formats and need to answer specific questions, and it's with these expectations of precise structure that archetypes can prove to be even more useful.
+
+For the sake of writing reviews, for instance, we could define the structure of a review beneath the front-matter of the `review.md` file:
+
+```markdown
++++
+manufacturer = ""
+model = ""
+price = ""
+releaseDate = ""
+rating = ""
 +++
+
+## Introduction
+
+## Sample Photos
+
+## Conclusion
+```
+
+Then, whenever we use the `hugo new` command to create a new review, not only will the default front-matter be copied into the newly created Markdown document, but the body of the `review.md` archetype will also be copied.
+
+To take this further though — and to ensure authors on multi-author websites are on the same page about how content should be written — we could include notes and reminders within the archetype:
+
+```markdown
++++
+manufacturer = ""
+model = ""
+price = ""
+releaseDate = ""
+rating = ""
++++
+
+## Introduction
+
+<!--
+  What is the selling point of the camera?
+  What has changed since last year's model?
+  Include a bullet-point list of key features.
+-->
+
+## Sample Photos
+
+<!-- TODO: Take at least 12 photos in a variety of situations. -->
+
+## Conclusion
+
+<!--
+  Is this camera worth the money?
+  Does it accomplish what it set out to achieve?
+  Are there any specific groups of people who should/shouldn't buy it?
+  Would you recommend it to a friend?
+  Are there alternatives on the horizon?
+-->
+
+```
+
+That way, each time we generate a new content file, we have a series of handy notes to push us closer to a piece of writing that's suitable for publishing.
+
+(If you're wondering why the notes are wrapped in the HTML comment syntax, it's to ensure that these notes won't appear inside the preview window of whatever Markdown editor the author happens to be using. The syntax isn't strictly necessary though.)
+
+This is still a fairly simple example, but if your content usually contains a variety of components — headings, bullet-points, images, [short-codes](/extras/shortcodes/), etc — it's not hard to see the time-saving benefits of placing these components in the body of an archetype file.
+
+## Theme Archetypes
+
+Whenever you generate a content file with the `hugo new` command, Hugo will start by searching for archetypes in the `archetypes` directory, initially looking for an archetype that matches the content's section and falling-back on the `default.md` archetype. If no archetypes are found in this directory, Hugo will continue its search in the `archetypes` directory of the currently active theme. In other words, it's possible for themes to come packaged with their own archetypes, ensuring that users of that theme format their content files with correctly structured front-matter.
+
```

Comment:
Hate to be a pain on this, but I think it's easy to get tripped up on default behavior. So, perhaps, after "and falling-back on the `default.md` archetype" you can say something like "if one is present" because if there's a default archetype in the top-level directory, any archetypes that exist only in the theme will be ignored.